### ISSUE TITLE
ENVI License : to fix the problem on the Adwaiseo computational server

### ISSUE THREAD

**Issue by wasdi-sysadmin-br** [fadeoutsoftware/WASDI#496]
On the Adwaiseo > computational node, when we try to reach the CIMA server we have a problem:
- the connection is seen on CIMA side
- but it seems the answer is blocked

Then it works with the local license server (172.17.0.1) but not with the remote ip (192...).

To work with ADWIAOSE to fix the problem

**Comment by wasdi-sysadmin-br** [comment]
**Update**

I started an analysis to try to understand the problem and I have a huge doubt about a network issue.
I shared my analysis with Paolo and I will wait his answer/opinion

**Comment by wasdi-sysadmin-br** [comment]
**Update**

The problem was identified and fixed.
It was not my idea of the 18/02/2022.

The problem was a time drift on the server: this server was 1m30 in the future.
This time drift introduced a problem to work with S3 object storage.
I installed Chrony + I fixed the hour:
  - since the S3 object storage problem is fixed
  - it fixed the problem of license
  
  This ticket can be tested + closed.
